src/main/java/frc/robot/commands/Shoot.java

```diff
@@ -37,18 +37,18 @@ public void execute() {
         // If the flywhheel is up to speed and the condition is true
         if (shooter.isFlywheelUpToSpeed() && shootCondition.get()) {
             indexer.activateConveyor();
-            indexer.activateGateWheel();
+            indexer.activateGateWheelForShooting();
```

Comment:
What does activate mean? Are we running those motors or are they just turning on? Is this common terminology FIRST/ WPI uses in their libraries to turn something on?